client/components/email-verification/email-unverified-notice.jsx

```diff
@@ -1,6 +1,8 @@
 /**
  * External dependencies
  */
+import PropTypes from 'prop-types';
+
 import * as React from 'react';
```

Comment:
This is _weird_. It should just be `import React from 'react';`. Want to include that and clean up the preceeding empty line? 🙇